page_objects/registration.page.js

```diff
@@ -0,0 +1,46 @@
+let BasePage = require("./base.page");
+let Button = require("../elements/button.element");
+let Input = require("../elements/input.element");
+
+
+let regEmailInputLocator = "#reg-form [type='email']";
+let nickInputLocator = "#reg-form [type='text']";
+let regPassInputLocator = "#reg-form [type='password']";
+let registrationButtonLocator = "#submit-button";
+
+class RegistrationPage extends BasePage {
+    getRegEmailInput() {
+        return new Input(element(by.css(regEmailInputLocator)), "Email Input");
+    }
+
+    getNickInput() {
+        return new Input(element(by.css(nickInputLocator)), "Nick name Input");
+    }
+
+    getRegPassInput() {
+        return new Input(element(by.css(regPassInputLocator)), "Password Input");
+    }
+
+    getRegistrationButton() {
+        return new Button(element(by.css(registrationButtonLocator)), "Registration Button");
+    }
+
+    async register(password) {
+        let timeStamp = Math.floor(Date.now() / 1000);
+        let regEmail = `autotshvets+${timeStamp}@gmail.com`;
```

Comment:
please move to test case. Re-do `register` method to work with to two parameters `email` and `password`

Reply:
moved  *let timeStamp = Math.floor(Date.now() / 1000);* to spec
added *email* and *nick* parameters to register method, because nick is required field and should be unique